tests/context.py

```diff
@@ -0,0 +1,5 @@
+import os
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join(
```

Comment:
I really don't like the fact that we are adding to sys.path.. is this really the only way to get unittest to see the code?
maybe some folder restructure would not need this? i need to see how i did it in the past but I am fairly sure it should not be needed..

Reply:
Agreed, removed, updated folder structures and added `__init__.py` files to hexgame folder and all the subfolders.